spec/services/user_service_spec.rb

```diff
@@ -161,6 +161,15 @@
       @comment_to_transfer =      create(:comment,                   user: @user_to_kill)
     end
 
+    context "admin submits same user to both keep and kill" do
+      it "does not kill the user" do
+        UserService.merge_users(user_to_keep: @user_to_keep, user_to_kill: @user_to_keep)
+
+        @user_to_keep.reload
+        expect(@user_to_keep.persisted?).to be true
```

Comment:
previously if `user_to_keep == user_to_kill` then the user would be destroyed ... this is how one Jess died ...